examples/run/cpu/seeding_example.cpp

```diff
@@ -95,9 +158,43 @@ int seq_run(const traccc::seeding_input_config& i_cfg,
 
         auto seeds = sa(spacepoints_per_event);
 
-        /*----------------
-             Statistics
-          ---------------*/
+        /*----------------------------
+           Track Parameter Estimation
+          ----------------------------*/
+
+        auto params = tp(spacepoints_per_event, seeds, readOut.modules,
+                         {0.f, 0.f, finder_config.bFieldInZ});
+
+        // Run CKF and KF if we are using a detray geometry
+        track_candidate_container_types::host track_candidates;
+        track_state_container_types::host track_states;
+
+        if (common_opts.run_detray_geometry == true) {
```

Comment:
How is the track finding and fitting run without detray? Or let me ask differently: Should the finding and fitting really go into the seeding example?
```suggestion
        if (common_opts.run_detray_geometry) {
```